specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/config/cache.options.parameterized.topic.validate.yaml

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+name: test
+catalogs:
+  test0:
+    type: test
+    options:
+      id: 1
+      schema: |
+        {
+          "fields": [
+            {
+              "name": "id",
+              "type": "string"
+            },
+            {
+              "name": "status",
+              "type": "string"
+            }
+          ],
+          "name": "Event",
+          "namespace": "io.aklivity.example",
+          "type": "record"
+        }
+bindings:
+  app0:
+    type: kafka
+    kind: cache_client
+    routes:
+      - exit: cache0
+        when:
+          - topic: test.*
+  cache0:
+    type: kafka
+    kind: cache_server
+    options:
+      topics:
+        - name: test.{id}
```

Comment:
We probably need to change `[^/]+` to `.+` for named parameters in topic such as `{id}`.

For example, if this was `test.{id}.control` then the matcher applied to `test.id1/0.control` would match `id` to `id0` instead of `id0/1` and then fail to match because the remainder would be just `.control` instead of `/0.control` as the (incorrect) matcher requires.